src/runtimes/go/index.ts

```diff
@@ -7,7 +7,7 @@ import { SourceFile } from '../../function'
 import { cachedLstat, cachedReaddir, FsCache } from '../../utils/fs'
 import { nonNullable } from '../../utils/non_nullable'
 import { detectBinaryRuntime } from '../detect_runtime'
-import { FindFunctionsInPathsFunction, Runtime, ZipFunction } from '../runtime'
+import { FindFunctionsInPathsFunction, GetFunctionAtPathFunction, Runtime, ZipFunction } from '../runtime'
```

Comment:
What's your recommendation? Rename `GetFunctionAtPathFunction` to `FindFunctionInPathFunction`?
The idea behind *Get* was to convey that it doesn't do any complex *finding*, but only parses the one path it's been given. If you read that otherwise / don't think that distinction is worth the inconsistency, I'm happy to defer.

Reply:
But isn't the complex finding taking place in [`getFunctionAtPath`](https://github.com/netlify/zip-it-and-ship-it/blob/isc-on-listfunctions/src/runtimes/node/finder.ts#L54-L75)? That's where we're filtering out runtime-specific paths like `node_modules`, distinguishing files from directories, figuring out the various allowed filenames and extensions (via calling `getMainFile`), etc.

In [`findFunctionsInPaths`](https://github.com/netlify/zip-it-and-ship-it/blob/isc-on-listfunctions/src/runtimes/node/finder.ts#L29-L52) we're just taking the results from `getFunctionAtPath` and sorting them?

Reply:
I somehow skimmed over that bit. falls in line with me naming that parameter `mainFile` instead of `path`. will rename to `FindFunctionInPathFunction`.

Reply:
implemented in https://github.com/netlify/zip-it-and-ship-it/pull/899/commits/4048c42aa618c193d278f7f276544dc412ea0fd5